edx_django_utils/monitoring/internal/middleware.py

```diff
@@ -476,11 +476,19 @@ def __init__(self, get_response):
     def __call__(self, request):
         response = self.get_response(request)
 
+        # Disable the middleware if flag isn't enabled
+        if not self._is_enabled():
+            return response
```

Comment:
If we want to entirely disable the middleware when the switch is disabled, we can `raise MiddlewareNotUsed()` (in `__init__` rather than `__call__`) to have Django remove the middleware from the list.

However, if we want the ability to do a percentage rollout (or other per-request decision), then this placement is correct, and I would just have a nit-picking comment of wanting to replace `Disable` in the comment with `Bypass` or `Skip`.